apps/game/server/players/player.service.ts

```diff
@@ -240,9 +240,9 @@ class _PlayerService {
    * @param src - Source of player being unloaded
    **/
   async handleUnloadPlayerEvent(src: number) {
-    await this.clearPlayerData(src);
+    await this.clearPlayerData.bind(this, src);
 
-    this.deletePlayerFromMaps(src);
+    this.deletePlayerFromMaps.bind(this, src);
```

Comment:
Totally forgot about this, the error was probably just fixed because the function is not called anymore. It is a different issue I suppose, you are right.